spaceship/lib/spaceship/portal/portal_client.rb

```diff
@@ -142,7 +143,7 @@ def update_service_for_app(app, service)
     end
 
     def associate_groups_with_app(app, groups)
-      ensure_csrf
+      ensure_csrf(Spaceship::AppGroup)
```

Comment:
Did you try using the unqualified class name here? Or is the FQN just preferable?